planetarium/NineChronicles
# Implement sub recipe view of recipes and connect action.

### Description

SSIA

### How to test

1. Check if sub recipes and options are displaying properly when crafting.
2. Check if equipments and consumables can be crafted.
3. Set `isEditor` variable to `false` in `GameConfig.cs` and check if recipe unlocking works properly.
